corpus/type-fields.txt

```diff
@@ -141,27 +141,27 @@ class A {
         (variable_declaration
           (function_pointer_type
             (function_pointer_parameter (predefined_type))
-            (function_pointer_parameter (predefined_type)))
+            (function_pointer_return_type (predefined_type)))
           (variable_declarator (identifier))))
       (field_declaration
         (variable_declaration
           (function_pointer_type
             (function_pointer_parameter
               (function_pointer_type
-                (function_pointer_parameter (predefined_type))
-                (function_pointer_parameter (predefined_type))))
-            (function_pointer_parameter
+                (function_pointer_parameter (parameter_modifier) (predefined_type))
+                (function_pointer_return_type (predefined_type))))
```

Comment:
Both types appear as FunctionPointParameter in Roslyn, it would be nice to align a little closer here, some options might be:

1. Alias `function_pointer_return_type` so it appears as a `function_pointer_parameter` 
2. Rename `function_pointer_return_type` to `function_pointer_parameter_return_type` (or something that keeps parameter in there)

Thoughts?

Reply:
Option 1 (aliasing) sounds good. I did it in https://github.com/tree-sitter/tree-sitter-c-sharp/pull/251/commits/72b525e30ee8665cae8b92b2585ab9c727a85a28.